### DISCUSSION TITLE
第4节课笔记(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#224] [笔记]
请提交笔记链接 Please submit the link to the notes.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/678212137

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/4-XTuner-78ed9b40fd504a88adddd9c81ac63c5b

**Comment by HaroldChen19** [comment]
https://github.com/HaroldChen19/InternLM-tutorial/blob/master/Lesson_4/Note.md

**Comment by jayjayhust** [comment]
https://github.com/jayjayhust/InternLM-Tutorial/blob/main/Lesson4.md

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/679314163
作业与笔记是同一个连接

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/679181675
https://zhuanlan.zhihu.com/p/679224539

**Comment by YongXie66** [comment]
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/lesson4.pdf

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679707416

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/4
